src/components/form/fields/string/String.tsx

```diff
@@ -25,26 +26,24 @@ export const FormStringField = ({
   ...rest
 }: IStringFormFieldProps) => {
   // When input value changes
-  const handleChange = useCallback((event: ChangeEvent<HTMLInputElement>): void => {
-    event.persist();
-
+  const handleChange = (event: ChangeEvent<HTMLInputElement>): void => {
     onChange(event.target.value?.toString(), event);
```

Comment:
is the event going to persist if you access the `value` here?

Reply:
![image](https://github.com/qoretechnologies/toolkit-react/assets/32803395/77640c0a-7896-4035-9086-41b8deaf3dee)

event.persist is kept only to make react backward compatible as it doens't do anything, you can find more details here https://legacy.reactjs.org/docs/events.html

Reply:
Ah, great, I did not know that it's no longer needed!